.github/pull-request-template.md

```diff
@@ -0,0 +1,38 @@
+<!--
+Thanks for contributing to Karpenter Core! Before making major changes, please read karpenter.sh/docs/contributing/design-guide
+-->
+
+<!-- Please follow the guidelines at https://www.conventionalcommits.org/en/v1.0.0/ and use one of the following in your title:
+feat:            <-- New features that require a MINOR version update
+fix:             <-- Bug fixes that require at PATCH version update
+docs:            <-- Documentation change that does not impact code
+chore:           <-- Metadata changes such as dependency update or configuration files
+test:            <-- Test changes that do not impact behavior
+perf:            <-- Code changes that improve performance but do not impact behavior
+BREAKING CHANGE: <-- Include if your change includes a backwards incompatible change.
+-->
+
+Fixes # <!-- issue number -->
+
+**Description**
+
+**How was this change tested?**
+
+*
+
+**Does this change impact docs?**
+- [ ] Yes, PR includes docs updates
+- [ ] Yes, issue opened: # <!-- issue number -->
+- [ ] No
+
+**Release Note**
```

Comment:
This part feels useless to me, we don't even use it at present

Reply:
You're right, removed